### PR TITLE
chore: update demo app to use shim

### PR DESCRIPTION
**Pull request by ashleyryan**
switch sidenav to vertical nav

Signed-off-by: Ashley Ryan <asryan@vmware.com>

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

**Review by bbogdanov — APPROVED** [review]

File: projects/demo/src/app/app.component.ts [hunk 1]
```diff
@@ -41,4 +41,12 @@ export class AppComponent {
     loadTextEditIconSet();
     loadTravelIconSet();
   }
+
+  onUseShimChange(e) {
```
Comment (ashleyryan) [inline projects/demo/src/app/app.component.ts:45]: Playing around with my demo url, this is doing a very bad job of keeping the body attribute in sync with the dropdown. 

I welcome suggestions for a more angular way of doing this.

Comment (kevinbuhmann) [reply projects/demo/src/app/app.component.ts:45]: I am looking into this. A directive might be the way to go here.

Comment (kevinbuhmann) [reply projects/demo/src/app/app.component.ts:45]: Removing the `*clrIfOpen` directive from the `clr-dropdown-menu` element makes it stop recreating the menu every time so the theme select value doesn’t revert back to "None" when reopening the menu.

Comment (ashleyryan) [reply projects/demo/src/app/app.component.ts:45]: Updated, thank you!

**Review by kevinbuhmann — APPROVED** [review]
I would split the nav change to its own commit, but it's fine either way.

**Pull request merged**